python/ray/tune/experimental/output.py

```diff
@@ -0,0 +1,585 @@
+from typing import List, Dict, Optional, Tuple, Any, TYPE_CHECKING
+
+import collections
+from dataclasses import dataclass
+import datetime
+from enum import Enum
+import numbers
+import numpy as np
+import os
+import pandas as pd
+from tabulate import tabulate
+import textwrap
+import time
+
+from ray._private.dict import unflattened_lookup
+from ray.air._internal.checkpoint_manager import _TrackedCheckpoint
+from ray.tune.callback import Callback
+from ray.tune.logger import pretty_print
+from ray.tune.result import (
+    AUTO_RESULT_KEYS,
+    EPISODE_REWARD_MEAN,
+    MEAN_ACCURACY,
+    MEAN_LOSS,
+    TIME_TOTAL_S,
+    TIMESTEPS_TOTAL,
+    TRAINING_ITERATION,
+)
+from ray.tune.experiment.trial import Trial
+
+if TYPE_CHECKING:
+    from ray.tune.stopper import Stopper
+
+# defines the mapping of the key in result and the key to be printed in table.
+# Note this is ordered!
+DEFAULT_COLUMNS = collections.OrderedDict(
+    {
+        MEAN_ACCURACY: "acc",
+        MEAN_LOSS: "loss",
+        TRAINING_ITERATION: "iter",
+        TIME_TOTAL_S: "total time (s)",
+        TIMESTEPS_TOTAL: "ts",
+        EPISODE_REWARD_MEAN: "reward",
+    }
+)
+
+VALID_SUMMARY_TYPES = {
+    int,
+    float,
+    np.float32,
+    np.float64,
+    np.int32,
+    np.int64,
+    type(None),
+}
+
+
+class AirVerbosity(Enum):
+    SILENT = 0
+    DEFAULT = 1
+    VERBOSE = 2
+
+    def __lt__(self, b):
+        return self.value < b.value
+
+
+try:
+    class_name = get_ipython().__class__.__name__
+    IS_NOTEBOOK = True if "Terminal" not in class_name else False
+except NameError:
+    IS_NOTEBOOK = False
+
+
+def get_air_verbosity() -> Optional[AirVerbosity]:
+    verbosity = os.environ.get("AIR_VERBOSITY", None)
+    if verbosity:
+        return AirVerbosity(int(verbosity)) if verbosity else None
+
+
+def _get_current_time():
+    return datetime.datetime.fromtimestamp(time.time())
```

Comment:
can you specify the return type? why not simply ``datetime.datetime.now()``?

Reply:
right, return type is a string instead of datetime type.

So something like "2023-03-22 13:22:34"